.buildkite/ftr_configs.yml

```diff
@@ -289,4 +289,4 @@ enabled:
   - x-pack/performance/journeys/web_logs_dashboard.ts
   - x-pack/performance/journeys/data_stress_test_lens.ts
   - x-pack/performance/journeys/ecommerce_dashboard_saved_search_only.ts
-  - x-pack/performance/journeys/ecommerce_dashboard_tsvb_gauge_only.ts
+  - x-pack/performance/journeys/saved_object_dashboard_list.ts
```

Comment:
Do we need to prefix with "saved_object"? Wouldn't "dashboard_listing_page" be more explicit? And saved object is an implementation details.